db/migrate/20150825073030_create_themes.rb

```diff
@@ -30,7 +30,10 @@ def change
       t.integer :background_text, limit: 4, default: 0x222222
       t.integer :background_muted, limit: 4, default: 0xBBBBBB
 
-      t.attachment :css
+      t.string :css_file_name
+      t.string :css_content_type
+      t.bigint :css_file_size
```

Comment:
This was a `t.integer` before -- please change it (and create a migration for that if it really should become a `bigint`)